### PR TITLE
Remove fallback

### PR DESCRIPTION
**Pull request by felics**
- Add sassdocs
- Add automatic grids

Implementing #38

### CODE REVIEW

**Comment by felics** [comment]
cc @renatocarvalho

File: scss/hagrid/_core.scss [hunk 1]
```diff
@@ -1,99 +1,74 @@
...
-    align-self: stretch;
+    @if $hagrid-flexibility {
+        @include flexibility;
+    }
     display: flex;
+    align-self: stretch;
     flex-flow: row wrap;
 
-    #{$hagrid-item-selector} {
+    > * {
```
Comment (felics) [inline scss/hagrid/_core.scss]: TODO: This should be a parameter passed to the mixin

**Comment by renatocarvalho** [comment]
Hey @felics Sorry for the delay. Here's what I have done:

- Tested the PR https://github.com/felics/hagrid/pull/41
- Tested on my Mac OS X El Captain (Chrome, Safari and Firefox). All good :+1: 
- Tested on Browserstack a few more browsers. Specially on IE 10 and IE 11. Those two are broken to me. :warning: 
- Reviewed the copy and the code changes. Looks good! :+1: 

I'm attaching the screenshots generated on Browserstack.

**Windows 8, IE 10**

win8_ie_10 0

**Windows 8, IE 11**

win8 1_ie_11 0

**Comment by felics** [comment]
thanks for the report! A quick test showed that [this bug in IE10-11](https://github.com/philipwalton/flexbugs#7-flex-basis-doesnt-account-for-box-sizingborder-box) seems to cause this issue, as the new PR uses flex-basis for width-calculations. I think I have to revert it to using width and adjust the auto-grid. IE at it again

 woohoo

**Comment by renatocarvalho** [comment]
haha, that image! Sad, but true. IE always blocking our way.

**Comment by felics** [comment]
@renatocarvalho This should fix it - I tested it against a IE10 & IE11 VM and didn't see anything unusual.

**Comment by renatocarvalho** [comment]
I'm going to test here again too.

**Comment by renatocarvalho** [comment]
@felics Did all the tests here and it looks good! Liked the new colors for the grid tests. :shipit:

**Comment by felics** [comment]
@renatocarvalho i'll push a last version of the v2 grid today (mainly cleaning up the code with stylelint)  and remove the flexibility stuff from v3, as you might as well use v2 in the case of needing legacy support. After that, the only thing left to do before merging is adding a note about installing v2.x via npm/bower

**Comment by renatocarvalho** [comment]
@felics Sounds good :+1:

**Pull request merged**